config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
We had N types before, and now have N*2 types. But the concept of registred type existed already. And the concept of registred type is only a small subset of mutants overall complexity presented in the both scores.

So a ~25% increase of a global score does not map quite right to the amount of complexity added.